functions.php

```diff
@@ -1,16 +1,13 @@
 <?php
 
-function eachCharacter($cartoons)
-{
-    foreach ($cartoons as $character) {
-        $characterImageElement = '<img src="' . $character['image'] . '" alt="Picture of' . $character['character_name'] . '">';
-        $characterNameElement = $character['character_name'];
-        $characterTvShowElement = $character['name'];
-        $characterIqElement = $character['IQ'];
-        $characterHtmlElement = '<div class="container">' . $characterImageElement . "<h2>Name: " . $characterNameElement . '</h2>' . "<h2>TV Show: " . $characterTvShowElement . '</h2>' . "<h2>IQ: " . $characterIqElement . '</h2>' . '</div>';
-        echo $characterHtmlElement;
-    };
-}
+/*
+ * Connects to the database, setting the attribute and executing the query by fetching all the data
+ *
+ * @param $collectionName refers to the database name, referenced through the function connect_db on the index.php file
+ *
+ * @return the new PDO, setting up the connection between the php file and the database
+ *
+ */
 
 function connect_db($collectionName)
 {
```

Comment:
You could consider changing this function name to connect_db_return_collection because it does more than connect to the db at the moment.